resources/views/components/filter/options.blade.php

```diff
@@ -37,41 +30,17 @@ class="focus:tw-ring-0 focus:tw-ring-offset-0 focus:tw-outline-none tw-form-chec
                     type="checkbox"
                     :key="option.id"
                     :id="option.id"
-                    @change="$emit('change', $event)"
                     :value="option.value"
-                    :name="filterName"
                     :checked="option.checked"
+                    @change="{{ $attributes->get('v-on:change') }}"
                 />
                 <span class="tw-font-normal tw-text-base"
-                    >{{ option.value }} <span class="tw-font-semibold">({{ option.count }})</span>
+                    >@{{ option.value }} <span class="tw-font-semibold">(@{{ option.count }})</span>
                 </span>
             </label>
         </div>
     </div>
-</template>
-
-<script>
-export default {
-    props: {
-        filterName: String,
-        placeholder: String,
-        selectedValues: Array,
-        filter: Array,
-    },
-    data() {
-        return {
-            search: '',
-        }
-    },
-    computed: {
-        filteredOptions() {
-            return this.filter
-                .filter((item) => (this.search ? item.value.includes(this.search) : true))
-                .map((option) => ({
-                    ...option,
-                    checked: (this.selectedValues || []).includes(option.value),
-                }))
-        },
-    },
-}
-</script>
+</search-controller>
+<x-filter.reset_button class="tw-hidden md:tw-flex tw-gap-1 tw-mb-6 tw-mt-5" @click="{{ $attributes->get('v-on:reset') }}">
```

Comment:
The `tw-gap-1` class probably belongs inside the component
```suggestion
<x-filter.reset_button class="tw-hidden md:tw-flex tw-mb-6 tw-mt-5" @click="{{ $attributes->get('v-on:reset') }}">
```

Reply:
Also, you're overriding `tw-flex` from inside the component with `tw-hide`. I know it works, but to me it is a bit flaky, because it probably depends on the order of the classes as they are applied. I'd consider using a wrapper and applying the hiding classes on that.